ArduPlane/quadplane.cpp

```diff
@@ -3583,4 +3598,13 @@ bool QuadPlane::show_vtol_view() const
     return show_vtol;
 }
 
+// return the tailsitter.transition_angle_vtol value if non zero, otherwise returns the tailsitter.transition_angle_fw value.
+AP_Int8 QuadPlane::get_tailsitter_transition_angle_vtol() const
```

Comment:
this should return a `Int8_t`

Reply:
followed existing declaration in structure....why change it?